recipe/meta.yaml

```diff
@@ -88,8 +88,8 @@ outputs:
         - dotnet-runtime ={{ runtime_version }}
     test:
       commands:
-        - dotnet --version
-    about:
+        - dotnet --version                   # [not aarch64]
+        - test -f $PREFIX/lib/dotnet/dotnet  # [aarch64]
```

Comment:
This is because conda is downloading ICU targeted to linux_64, since it is hosting the test in that environment, rather than having the ICU package built for aarch64 that dotnet expects.